.changeset/purple-falcons-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'@tokens-studio/sd-transforms': minor
+---
```

Comment:
```suggestion
---
'@tokens-studio/sd-transforms': patch
---
```
because <1.0.0 a minor would be for breaking change, and the addition of this transform is by itself not breaking, so prefer a patch there.

However, the changing of the name of the CSS typography transform (both the exported function name and the SD transform name as it's registered) is breaking, so can you create a 2nd changeset, using `minor` and maybe this message:
```
**BREAKING**: Renames typography transform to clarify it's transformed to CSS-based format.
```

Changeset will basically take both the changesets (patch one and minor one) and process them correctly in the changelog and release a minor version when done.